playbooks/lb_down_monitoring_setup.yaml

```diff
@@ -17,12 +17,31 @@
   tasks:
     - name: Register build result
       set_fact:
+        lb_down_instance_ip: "{{ tf_output.outputs['lb_down_instance_fip'].value }}"
         lb_down_node_ips: "{{ tf_output.outputs['lb_down_ecs_ips'].value }}"
+        lb_control_instance_ip: "{{ tf_output.outputs['lb_ctrl_ip'].value }}"
+        key_name: "{{ key_name }}"
+        local_private_key: "{{ local_private_key }}"
+    - name: Register controller node
+      add_host:
+        name: "{{ lb_control_instance_ip }}"
+        groups: control
+        ansible_ssh_user: linux
+        ansible_ssh_private_key_file: "{{ local_private_key }}"
+        ansible_ssh_common_args: >
+          -o UserKnownHostsFile=/dev/null
+          -o ProxyCommand='ssh -W %h:%p
+          -q linux@{{ controller_public_ip }}
+          -i {{ local_private_key }}
+          -o StrictHostKeyChecking=no
+          -o UserKnownHostsFile=/dev/null'
+          -o PasswordAuthentication=no
+
     - name: Register nodes
       add_host:
         name: "{{ item }}"
-        groups: "gatewayed"
-        ansible_ssh_user: "linux"
+        groups: gatewayed
+        ansible_ssh_user: linux
         ansible_ssh_private_key_file: "{{ local_private_key }}"
```

Comment:
Can `ansible_ssh_user` and `ansible_ssh_private_key_file` be a part of inventory definition of `gatewayed` group?

Reply:
hmm, ok